.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ocaml.packageManager": "esy"
```

Comment:
are you using the ocamllabs extension? How good is it :) ?

Reply:
Except for the bug where I can't run `esy` in the terminal it's great 😂